src/java/fr/paris/lutece/portal/service/security/LuteceUserCacheService.java

```diff
@@ -60,7 +60,7 @@ public static LuteceUserCacheService getInstance( )
     {
         if ( _instance == null )
         {
-            synchronized( LuteceUserCacheService.class )
+            synchronized ( LuteceUserCacheService.class )

```

Comment:
same remark here for synchronisation scope

Reply:
Corrected in another commit of the same PR.